src/views/Dashboard.vue

```diff
@@ -41,7 +41,7 @@
       </v-flex>
     </v-layout>
     <v-divider />
-    <v-layout row wrap>
```

Comment:
Hi @kylerits 

This has already been merged to master - https://github.com/cylc/cylc-ui/blob/46c757b92e9c344f2dcbc9c2ad1ca348dcc2f195/src/views/Dashboard.vue#L44

So it shouldn't appear here on your PR. You need to rebase onto master, dropping the commit where you added this file. Leaving only `package.json` and `package-lock.json` as changes here :slightly_smiling_face: